### DISCUSSION TITLE
Registration guidelines

### DISCUSSION THREAD

**Discussion by shu-bamma** [larics/icuas22_competition#31] [Q&A] [answered]
@RobertMilijas
Do we need to register the team officially for the competition somewhere? I could only find the PRELIMINARY REGISTRATION form on the website. ([link](http://www.uasconferences.com/2022_icuas/uav-competition-expression-of-interest/))
Was filling this form before the simulation phase necessary to enter the competition?

**Comment by fpetric** [comment, marked as answer]
@shu-bamma 
if you haven't already, please use the preliminary registration form. As you can see in #32, we are still preparing the infrastructure and instructions for submission of team application forms. We will announce the details soon. Stay tuned for announcements here!
